.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Chat 👾
+    url: https://astro.build/chat
+    about: Our Discord server is active, come join us!
+  - name: Support 💁
+    url: https://astro.build/chat
+    about: 'This issue tracker is not for support questions. Join us on Discord for assistance!'
```

Does this link need listing twice, or can we combine these into one? Maybe something along the lines of:
```suggestion
  - name: Support & Community 💁
    url: https://astro.build/chat
    about: 'This issue tracker is not for support questions. Our Discord server hosts an active community of Astro users. Come join us to discuss Astro or find assistance!'
```

---

Sure! it's no problem